nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/EncryptContent.java

```diff
@@ -138,6 +141,17 @@
             .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
             .sensitive(true)
             .build();
+
+    public static final PropertyDescriptor PGP_SYMMETRIC_ENCRYPTION_CIPHER = new PropertyDescriptor.Builder()
```

Comment:
I already tried to add a custom validation case (and a test), however, I didn't commit that because, since the default value (and not supporting any kind of EL), I was not able to reach an invalid configuration. 

Should we anyway add the custom validate method?

Reply:
I added a null check in the custom validation for the `PGP_SYMMETRIC_ENCRYPTION_CIPHER` as well as a couple of test cases, however, I am not sure about the custom validation, what do you think about it?